cocos/scene-graph/node.ts

```diff
@@ -336,11 +336,7 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
         }
     }
 
-    /**
-     * @engineInternal this is engineInternal for it doesn't have side effect of setting parent.
-     */
-    @serializable
-    public _parent: this | null = null;
+    protected _parent: this | null = null;
```

Comment:
Why remove the `@serializable` decorator ?

Reply:
It is a mistake. Add it back.